src/index.html

```diff
@@ -0,0 +1,273 @@
+        <nav>
+          <button class="navbar__menu-button" type="button">
+            <img src="./assets/icons/menu.svg" alt="Menu icon" />
+          </button>
+          <div class="navbar__links-list-container">
+            <button class="navbar__close-button" type="button">
+              <img src="./assets/icons/chevron-left.svg" alt="Close icon" />
+            </button>
+            <ul class="navbar__links-list">
+              <li class="navbar__links-list-item">
+                <a href="./about.html">About</a>
+              </li>
+              <li class="navbar__links-list-item">
+                <a href="./about.html">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+        <button class="navbar__cta-button" type="button">
+          Get Your Ticket
+        </button>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="event">
+        <div class="container">
+          <div class="event__overtitle">"Hello! JavaScript Lovers"</div>
+          <h1 class="event__title">
+            Global JavaScript Conference <br />
+            2022
+          </h1>
+          <p class="event__description">
+            JavaScript is #1 on demand programming language. Meet the ones who
+            shape out the now and the future of the prospering nation.
+          </p>
+          <div class="event__date">JUNE, 16th ~ 20th</div>
+          <div class="event__location">
+            @ Gedempt Hamerkanaal 231 1021 KP Amsterdam
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section section--dark">
+      <div class="container">
+        <h2 class="section__title">Main Program</h2>
+        <div class="program">
+          <div class="program__card">
+            <div class="program__card-icon">
+              <img src="./assets/icons/login.svg" alt="Login icon" />
+            </div>
+            <div class="program__card-title">Opening</div>
+            <div class="program__card-text">
+              A small opening ceremony, to welcome everybody, and give a briefe
+              introduction to the conference.
+            </div>
+          </div>
+          <div class="program__card">
+            <div class="program__card-icon">
+              <img src="./assets/icons/speaker.svg" alt="Speaker icon" />
+            </div>
+            <div class="program__card-title">Lecture</div>
+            <div class="program__card-text">
+              Listen to the speakers and inreach your knowledge.
+            </div>
+          </div>
+          <div class="program__card">
+            <div class="program__card-icon">
+              <img src="./assets/icons/info.svg" alt="Info icon" />
+            </div>
+            <div class="program__card-title">Q&A</div>
+            <div class="program__card-text">
+              If you have a question, you will have the opportunity to ask the
+              speaker.
+            </div>
+          </div>
```

Comment:
- [ ] Kindly make sure to align all the text, images and paragraphs as suggested in the loom video.